### DISCUSSION TITLE
why is sat sat 2099999997683968 uncommon?

### DISCUSSION THREAD

**Discussion by miitchmatch** [ordinals/ord#2319] [Q&A] [answered]
Hello there, I am writing a thesis regarding ordinals and need your help regarding the rarity of sat 2099999997683968 (name: hwz). 
According to ordinals.com it is uncommon, if I correctly interpret the degree 5°203968′1024″0‴ it’s because it’s the first sat in the block!?! 
What confuses me is, that the next and previous sats are also uncommon and also end the degree ″0‴.
https://ordinals.com/sat/2099999997683968

I know this sat will be mined way in the future, but since hwz is the name of my school, i'd like to use it as an example...

**Comment by raphjaph** [comment, marked as answer]
They are all the first sat of consecutive blocks. These are some of the last sats to be mined and since the subsidy decreases exponentially over the next one hundred years at that point there will only be a 1 sat subsidy per block.

**Reply by miitchmatch** [reply]
Thanks a lot for your feedback. Didn‘t really understand block rewards and halvings until now! 
but i will probably go with a longer word for my thesis!;)
